cpp/benchmarks/CMakeLists.txt

```diff
@@ -175,6 +175,16 @@ ConfigureBench(COPY_IF_ELSE_BENCH copying/copy_if_else.cpp)
 # * transpose benchmark ---------------------------------------------------------------------------
 ConfigureBench(TRANSPOSE_BENCH transpose/transpose.cpp)
 
+# ##################################################################################################
+# * apply_boolean_mask benchmark ------------------------------------------------------------------
+ConfigureBench(APPLY_BOOLEAN_MASK_BENCH stream_compaction/apply_boolean_mask.cpp)
+
+# ##################################################################################################
+# * tpch benchmark --------------------------------------------------------------------------------
+ConfigureNVBench(TPCH_Q1 tpch/q1.cpp)
+# ConfigureNVBench(TPCH_Q5 tpch/q5.cpp) ConfigureNVBench(TPCH_Q6 tpch/q6.cpp)
+# ConfigureNVBench(TPCH_Q9 tpch/q9.cpp) ConfigureNVBench(TPCH_Q10 tpch/q10.cpp)
```

Comment:
Are these supposed to be commented out?

Reply:
These were uncommented in the latest commit . Sorry